src/lib/pages/pools/components/unsupportedSection/index.tsx

```diff
@@ -46,11 +47,16 @@ export const UnsupportedSection = ({
     },
   });
   const { poolTypeValue, keyword, isSuperfluidOnly } = watch();
+  const search = useMemo(
+    () => (!keyword || isPositiveInt(keyword) ? keyword : `{"${keyword}"}`),
+    [keyword]
+  );
```

Comment:
I think you can remove this since the logic is similar to the one in `usePoolListCountQuery` queryFn, configure that one instead